x/authz/client/cli/tx.go

```diff
@@ -173,10 +173,22 @@ Examples:
 	cmd.Flags().String(FlagSpendLimit, "", "SpendLimit for Send Authorization, an array of Coins allowed spend")
 	cmd.Flags().StringSlice(FlagAllowedValidators, []string{}, "Allowed validators addresses separated by ,")
 	cmd.Flags().StringSlice(FlagDenyValidators, []string{}, "Deny validators addresses separated by ,")
-	cmd.Flags().Int64(FlagExpiration, time.Now().AddDate(1, 0, 0).Unix(), "The Unix timestamp. Default is one year.")
+	cmd.Flags().Int64(FlagExpiration, 0, "Expire time as Unix timestamp. Set zero (0) for no expiry. Default is 0.")
```

Comment:
I've changed the default here to zero = no expiration time.
Maybe we would like to explicitly require the value? In that case:
* 0 would be not allowed
* -1 = no expire date

Reply:
I think 0 == no expiration time makes the most sense.

Alternatively we can do:
- 0 == no expiry
- default is -1
- all negative values are not allowed

Reply:
> Alternatively we can do: 0 == no expiry; default is -1

If we want to make the CLI param required, the I prefer -1 as _no expire_, this is because 0 would mean "default", and we don't want default (from the user perspective).

> all negative values are not allowed

Now, all values < `blocktime().unix()` are not allowed , except zero.

Reply:
i was thinking:
```suggestion
	cmd.Flags().Int64(FlagExpiration, -1, "Expire time as Unix timestamp. Set zero (0) for no expiry.")
```

But tbh -1 = no expire date sounds good to me too, if there's a help message.